src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/integration/bdd/BddSteps.kt

```diff
@@ -70,15 +72,15 @@ class BddSteps : En {
 
   init {
 
-    Before { scenario: Scenario ->
+    Before { _: Scenario ->
 
       offenderEventsClient.purgeQueue(PurgeQueueRequest(eventQueueUrl))
       calculationCompleteClient.purgeQueue(PurgeQueueRequest(calculationCompleteUrl))
 
       setupOauth()
-      val re = Regex("[^A-Za-z0-9]")
-      crn = re.replace(scenario.name, "").replace(" ", "")
-      setupData = SetupData(communityApi, assessmentApi, crn)
+      crn = UUID.randomUUID().toString().replace("-", "").substring(0, 7)
+      assessmentId = UUID.randomUUID().toString().replace("\\D+".toRegex(), "").substring(0, 11)
+      setupData = SetupData(communityApi, assessmentApi, mapOf("crn" to crn, "assessmentId" to "1$assessmentId"))
```

Comment:
It's a number in JSON so cannot have a leading 0

Reply:
bikeshedding but prepend a "1" to it?